Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -15857,6 +15858,17 @@ void WebPageProxy::setPresentingApplicationAuditToken(const audit_token_t& prese
 }
 #endif
 
+#if ENABLE(CONTENT_EXTENSIONS)
+void WebPageProxy::didExeedFrameNetworkUsage(WebCore::FrameIdentifier frameID, String host)
+{
+    auto& throttler = WebCore::ResourceMonitorManager::singleton().throttler();
+
+    auto granted = throttler.tryAccess(WTFMove(host));
+    if (granted)
+        send(Messages::WebPage::DidExeedFrameNetworkUsage(frameID));
```

Comment:
Is there a reason this is sending a message rather than defining this IPC as one that returns a value?

Reply:
Changed to use sendWithAsyncReply.